assets/components/paymentMethods/paymentMethods.jsx

```diff
@@ -19,6 +18,8 @@ type PropTypes = {
   lastName: string,
   error: ?string,
   payPalButtonExists: boolean,
+  stripeCallback: Function,
+  paypalCallback: Function,
 };
```

Comment:
Since everything is passed through as props, this doesn't need to be redux 'connect'ed any more.